nwbwidgets/test/test_ophys.py

```diff
@@ -132,10 +122,30 @@ def setUp(self):
         self.df_over_f = DfOverF(rrs)
 
     def test_show_two_photon_series(self):
-        assert isinstance(
-            TwoPhotonSeriesWidget(self.image_series2, default_neurodata_vis_spec),
-            widgets.Widget,
+        self.image_series2 = TwoPhotonSeries(
+            name="test_iS2",
+            data=np.random.randn(100, 5, 5),
+            imaging_plane=self.imaging_plane,
+            starting_frame=[0],
+            rate=1.0,
+            unit="n.a",
+        )
+        ts_wid = TwoPhotonSeriesWidget(self.image_series2, default_neurodata_vis_spec)
+        slider = [i for i in ts_wid.children if isinstance(i, widgets.IntSlider)][0]
```

Comment:
I don't like this way of getting the slider attribute. If we want to make the slider accessible outside the class (as it should be) then we should make it a class attribute and get it that way.